Code/GraphMol/ChemReactions/catch_tests.cpp

```diff
@@ -31,7 +31,7 @@
 
 using namespace RDKit;
 using std::unique_ptr;
-
+/*
```

Comment:
why is this test now disabled?

Reply:
Whups, it was a debugging aid for me, I've reenabled them.